.github/workflows/change-version.yml

```diff
@@ -0,0 +1,29 @@
+name: Change Version
+
+on:
+  create:
+     branches:
+      - 'release_v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF##*/})"
+      id: extract_branch
+    - name: Change the version number
```

Comment:
As discussed, there are still a few places where this script fails in the codebase. I would recommend we update those places in this PR so it can be fully automated and we don't have to worry about it any longer, i.e., either change the format to be consistent with all others or remove if we don't need.

I listed them on slide 2 bullet 8 in these release slides a while back: https://docs.google.com/presentation/d/16uzwIOAUvDrYibWMyrApBqqNvO8nEnr_vJYlX_W6gXg/edit?usp=sharing

Some are out of date (prob don't need to worry about Xcode or configure.ac)